docker-compose.yml

```diff
@@ -150,7 +150,79 @@ services:
       - traefik.docker.network=cryptlex-on-premise_backend
       - traefik.port=4200
 
-  release-server:
+ admin-portal:
```

Comment:
the indentation is incorrect which makes the file invalid. YAML is sensitive to indentation